app/controllers/users/emails_controller.rb

```diff
@@ -6,12 +6,14 @@ class Users::EmailsController < ApplicationController
   before_action :authorize_user
   before_action :redirect_to_account_page_if_acting_on_own_user, only: %i[edit]
 
+  helper_method :return_path
```

Comment:
There's a typo in the message for the commit titled "Adapt Users::EmailsController to work with API users". The message refers to `Users::NamesController` but I think it should be `Users::EmailsController`.

Reply:
Well spotted! Too much copy/pasting!

Reply:
Addressed in https://github.com/alphagov/signon/pull/2593/commits/7434b1abdc8fc842892f532cb7b3cc1d98d6c406